Sponsors.md

```diff
@@ -1,90 +1,91 @@
-* (optionally) any other item that can fit in the budget
+The financial contributions may be used by DevStaff for various purposes to support the community, e.g., providing food at a meetup, ordering swag to give out to community members, etc. 
+These activities are expensed in accordance to OpenCollective's policies, including submitting a receipt for transparency purposes.
 
-Here’s how the swag draw will work: 
-* 5 swag packs will be awarded at each meetup
-* Each pack costs 20 EUR
-* 100 EUR buys us 5 packs - i.e. awareness for you for ONE meetup. 
-* 200 EUR buys us 10 packs - i.e. awareness for you for TWO meetups. 
-* 400 EUR buys us 20 packs - i.e. awareness for you for FOUR meetups. 
+## Other ways to sponsor
+
+We provide a few other ways to sponsor DevStaff outside OpenCollective, that provides the advantage to a legal entity (e.g., a company) to be directly invoiced. Your contribution will be made aware to our community (e.g., during meetups), but note that it cannot be listed in our OpenCollective page.
+
+### Swag Sponsor
+
+Help the community members wear the DevStaff community name with pride! With this option, you can sponsor the purchase of promotional material with the DevStaff brand, in return for awareness of your own brand.  (Please note we do not currently offer a co-branding option, see below for awareness benefits). All material will be made available to community and org team members.
+
+Each swag pack may include a dev-friendly cool looking t-shirt, and/or "Raki" / "Tsikoudia" shot glasses, and/or any other item that can fit in the budget.
+
+Each swag pack will be awarded to lucky attendees through a lottery that will take place at the meetup. Your company will be publicly thanked for making this possible. 
+
+Get in contact with us to discuss swag pack opportunities. Please note that depending on the amount of swag packs, we will publicly thank you in a number of next meetups. For example:
+
+* 5 swag packs - i.e. awareness for you for ONE meetup. 
+* 10 swag packs - i.e. awareness for you for TWO meetups. 
+* 20 swag packs - i.e. awareness for you for FOUR meetups. 
 
 The list below describes the awareness benefits that our branding sponsor receives in return:
-* leaflets (can be left on table at entrance),
-* “Thank you” message by organizers during Meetup (not by company),
-* “Thank you” message during Swag Pack draw
-* “Thank you” mention in facebook page post.
+* leaflets (can be left on a table at entrance),
+* "Thank you" message by organizers during Meetup (not by company),
+* "Thank you" message during Swag Pack draw
+* "Thank you" mention in our social media.
 
 Note: the invoices will be issued directly to the sponsoring company by our own suppliers. 
 
-## DevStaff Laptop Stickers Sponsor
-No developer laptop is complete without laptop stickers. A “badge of honor” for many, a hipster trend for some… but the reality is that laptop stickers are highly sought after! 
+### DevStaff Laptop Stickers Sponsor
```

Comment:
Isn't this a SWAG too? maybe we could combine with the previous section (SWAG). What do you think?

Reply:
Merged